### PR TITLE
update to latest ES and SG 2.x versions

### PR DESCRIPTION
**Pull request by richm**
@lukas-vlcek ptal

### CODE REVIEW

File: tests/setup/sg_init.sh [hunk 1]
```diff
@@ -73,6 +73,7 @@ if [ "${IS_ES_SECURED:-}" = true ] ; then
     cp ${TMP_DIR}/search-guard-ssl/example-pki-scripts/transport/node-0-keystore.jks ${ES_HOME}/plugins/search-guard-2/sgconfig/transport-node-0-keystore.jks
     cd ${ACTUAL_DIR}
 
+    cp ${ES_CONF}/sg_*.yml ${ES_HOME}/plugins/search-guard-2/sgconfig
```
Comment (lukas-vlcek) [inline tests/setup/sg_init.sh]: Aren't SG config files installed automatically when the SG plugin is [installed](https://github.com/ViaQ/watches-cli/blob/2f58f73a4234ee3578bf490e79edce4231fb850c/tests/setup/sg_init.sh#L65) using `${ES_HOME}/bin/plugin install -b com.floragunn/search-guard-2/${SG_VER}` command?

Or these SG config files are different from those provided with SG by default?

File: tests/setup/sg_init.sh [hunk 2]
```diff
@@ -12,9 +12,9 @@ set -euxo pipefail
 # ----------------------------------------------
 
 export IS_ES_SECURED=${IS_ES_SECURED:-true}
-export ES_VER=${ES_VER:-2.4.1}
-export SG_VER=${SG_VER:-2.4.1.8}
-export SG_SSL_VER=${SG_SSL_VER:-2.4.1.16}
+export ES_VER=${ES_VER:-2.4.4}
+export SG_VER=${SG_VER:-2.4.4.10}
+export SG_SSL_VER=${SG_SSL_VER:-2.4.4.19}
```
Comment (lukas-vlcek) [inline tests/setup/sg_init.sh]: We have been running travis tests against [these versions](https://github.com/ViaQ/watches-cli/blob/2f58f73a4234ee3578bf490e79edce4231fb850c/.travis.yml#L21-L23) for some time now. So may be we can comment out travis tests agains [older versions](https://github.com/ViaQ/watches-cli/blob/2f58f73a4234ee3578bf490e79edce4231fb850c/.travis.yml#L18-L20)? We probably do not need travis run agains `4.2.2` version any loner, right?

Comment (richm) [inline tests/setup/sg_init.sh]: Right.

File: tests/setup/sg_init.sh (hunk 1 above)
Comment (richm) [inline tests/setup/sg_init.sh]: They are slightly different.  They give us the ability to test with a sg config more similar to openshift, if desired.

**Comment by lukas-vlcek** [comment]
Thanks @richm

**Pull request merged**